wolfssl/wolfcrypt/wc_port.h

```diff
@@ -487,6 +491,11 @@ WOLFSSL_API int wolfCrypt_Cleanup(void);
     #define XSEEK_END                VSEEK_END
     #define XBADFILE                 -1
     #define XFGETS(b,s,f)            -2 /* Not ported yet */
+    #define XVFPRINTF                vfprintf
```

Comment:
Instead of adding all these variations just do an `#ifndef XVFPRINTF` at bottom for all these.

Reply:
Addressed